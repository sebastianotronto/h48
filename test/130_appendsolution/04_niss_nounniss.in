U F B2
R' F D
0
1
rotation UF
