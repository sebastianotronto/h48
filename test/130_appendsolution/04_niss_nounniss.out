U F B2 (R' F D)
Number of solutions: 1
Shortest solution length: 6
Used bytes: 16
